ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -610,9 +607,25 @@ class JITModBuilder {
     AllocaInst *ActiveFrameVec = FailBuilder.CreateAlloca(
         ActiveFrameSTy,
         ConstantInt::get(PointerSizedIntTy, ActiveFrames.size()));
+
+    std::vector<Value *> LiveValues;
     for (size_t I = 0; I < ActiveFrames.size(); I++) {
       FrameInfo FI = ActiveFrames[I];
 
+      // Read live AOT variables from stackmap calls and match them to
+      // variables in the JIT so we they can be optimised.
```

Comment:
> so we they can be optimised

Should this be:

> So they can be deoptimised?

But even then, maybe it could be explained a little more clearly?

Reply:
Fixed in [3a22adb](https://github.com/ykjit/yk/pull/562/commits/3a22adbce65409477aa8f4e83c0b5e17f6da2045)